Docs: adding the GHS+RTC info. (#13185)
ckeditor/ckeditor5: packages/ckeditor5-html-support/docs/features/general-html-support.md

It is possible to add support for arbitrary styles, classes and other attributes to existing CKEditor 5 features (such as paragraphs, headings, list items, etc.). Most of the existing CKEditor 5 features can already be extended this way, however, some cannot yet. This includes e.g.: the `<ul>` and `<ol>` elements of the list feature (see: [#9917](https://github.com/ckeditor/ckeditor5/issues/9917)).

<info-box info>

> 	While the GHS feature is stable, if it is used in connection with {@link features/real-time-collaboration real-time collaboration} there may be problems with complex documents.

Reviewer: ```suggestion
	While the GHS feature is stable, some problems with complex documents may occur if it is used together with {@link features/real-time-collaboration real-time collaboration}.
```
